rooms/rooms.py

```diff
@@ -0,0 +1,5 @@
+class Rooms(object):
+    """This is the base class for office and livingspace class"""
+    def __init__(self, name):
+        self.name = name
+        self.total_space = 4
```

Comment:
Quick observation: I see that by default a Room has a capacity of 4. 

Do you think it would be a better design strategy to have this this property `total_space` declared only in the subclasses i.e Livingspace and Office? I say this because a Room in itself has no capacity; only an Office or a Livingspace has. While your approach works, I believe a cleaner OOP approach would be to have each class own properties that are directly related to it. 

You can even go further and make Room class abstract.

What do you think about that?